substra/cli/printers.py

```diff
@@ -343,6 +351,74 @@ class TraintuplePrinter(AssetPrinter):
     has_description = False
 
 
+class TraintuplePrinter(BaseTraintuplePrinter):
+    asset_name = 'traintuple'
+
+    list_fields = (
+        Field('Composite algo name', 'algo.name'),
+        Field('Status', 'status'),
+        Field('Perf', 'dataset.perf'),
+        Field('Tag', 'tag'),
+        Field('Compute Plan Id', 'computePlanID'),
+        Field('Head Model Key', 'head_model_key'),
+        Field('Trunk Model Key', 'trunk_model_key'),
```

Comment:
This looks like a mistake, traintuples don't have head/trunk model inputs.

Reply:
Actually it looks like the whole class is really a CompositeTraintuplePrinter, there maybe some cleanup to do here.